src/traces/parcoords/attributes.js

```diff
@@ -0,0 +1,188 @@
+                'of `[fromValue, toValue]` with finite numbers as elements.'
+            ].join(' ')
+        },
+        values: {
+            valType: 'data_array',
+            role: 'info',
+            description: [
+                'Dimension values. `values[n]` represents the value of the `n`th point in the dataset,',
+                'therefore the `values` vector for all dimensions must be the same (longer vectors',
+                'will be truncated). Each value must be a finite number.'
+            ].join(' ')
+        },
+        description: 'The dimensions (variables) of the parallel coordinates chart. 2..63 dimensions are supported.'
+    },
+
+    tickdistance: {
+        valType: 'number',
+        dflt: 50,
+        min: 32,
+        role: 'style',
+        description: 'The desired approximate tick distance (in pixels) between axis ticks on an axis.'
+    },
+
+    padding: {
+        valType: 'number',
+        dflt: 80,
+        min: 0,
+        role: 'style',
+        description: 'The desired space for displaying axis labels and domain ranges around the actual parcoords.'
+    },
+
+    line: extendFlat({},
+
+        colorAttributes('line'),
+
+        {
+            showscale: {
+                valType: 'boolean',
+                role: 'info',
+                dflt: false,
+                description: [
+                    'Has an effect only if `marker.color` is set to a numerical array.',
+                    'Determines whether or not a colorbar is displayed.'
+                ].join(' ')
+            },
+            colorbar: colorbarAttrs
+        },
+
+        {
+            contextopacity: {
+                valType: 'number',
+                dflt: 0.05,
+                min: 0,
+                max: 1,
+                role: 'style',
+                description: 'Opacity of the context lines, on a scale of 0 (invisible) to 1 (fully opaque).'
+            },
+
+            focusopacity: {
+                valType: 'number',
+                dflt: 1,
+                min: 0,
+                max: 1,
+                role: 'style',
+                description: 'Opacity of the focus lines, on a scale of 0 (invisible) to 1 (fully opaque).'
+            },
+
+            pixelratio: {
+                valType: 'number',
+                dflt: 1,
+                min: 0.25,
+                max: 4,
+                role: 'style',
+                description: 'Line rendering pixel ratio. A lower value yields faster rendering but blockier lines.'
+            },
+
+            focusalphablending: {
+                valType: 'boolean',
+                dflt: false,
+                role: 'style',
+                description: [
+                    'By default, the rendered lines are opaque. Setting it to `true` is necessary if opacity is needed.'
+                ].join(' ')
+            }
+        }
+    ),
+
+    blocklinecount: {
```

Comment:
Will do! 

As for original motive (very optional read; real solution in final paragraph): We don't currently _measure_ in runtime how fast the rendering is. On some hardware, you can render 10k lines in one `rAF` slice - in this case, the ideal `blocklinecount` is 10k. On another, only 1k etc. If `blocklinecount` is too low, then even on powerful hardware, the incremental rendering kicks in. If it's too high,  then it attempts to render all lines in one fell swoop - resulting in janky or frozen interactions.

I initially added runtime performance measurement. But later internal changes made it too expensive to run. 

The ideal solution is to gather runtime statistics on block rendering and tweak this cap real time in line with that, probably a few hrs of work, I'll put it on the backburner for now.